homeassistant/components/sonos/media_player.py

```diff
@@ -741,6 +753,21 @@ def remove_from_queue(self, queue_position: int = 0) -> None:
         """Remove item from the queue."""
         self.coordinator.soco.remove_from_queue(queue_position)
 
+    @soco_error()
+    def get_queue(self) -> list[dict]:
+        """Get the queue."""
+        queue: list[DidlMusicTrack] = self.coordinator.soco.get_queue(max_items=0)
+        retval: list[dict] = []
+        for track in queue:
+            track_item = {
+                ATTR_MEDIA_TITLE: track.title,
+                ATTR_MEDIA_ALBUM_NAME: track.album,
+                ATTR_MEDIA_ARTIST: track.creator,
+                ATTR_MEDIA_CONTENT_ID: track.get_uri(),
+            }
+            retval.append(track_item)
+        return retval
```

Comment:
This can be a list comprehension instead

Reply:
good point, simpler that way - thanks